#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-remove-gjdoc-inexistent-options.dpatch by Tiago Saboga <tiagosaboga@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: gjdoc fails to generate javadoc when it is given the -notimestamp
## DP: option.

@DPATCH@
diff -urNad libjna-java-3.0.2~/build.xml libjna-java-3.0.2/build.xml
--- libjna-java-3.0.2~/build.xml	2008-05-31 18:23:08.000000000 -0300
+++ libjna-java-3.0.2/build.xml	2008-05-31 18:23:09.000000000 -0300
@@ -472,7 +472,6 @@
       <group title="Unix Examples" packages="com.sun.jna.examples.unix"/>
       <group title="Windows Examples" packages="com.sun.jna.examples.win32"/>
 
-      <arg value="-notimestamp"/>
     </javadoc>
   </target>
 
